app/controllers/api/braintree_controller.rb

```diff
@@ -37,11 +38,15 @@ def subscription_options
     {
       amount: params[:amount].to_f,
       plan_id: '35wm',
-      payment_method_token: Payment.customer(params[:email]).default_payment_method.token
+      payment_method_token: default_payment_method_token
     }
   end
 
   def braintree
     PaymentProcessor::Clients::Braintree
   end
+
+  def default_payment_method_token
+    @token ||= ::Payment.customer(params[:email]).try(:card_vault_token)
```

Comment:
This might be something that the user wouldn't be able to handle (e.g. we failed to store the vault token / there was an error from braintree creating the user) or it could be that the user has entered a wrong e-mail address incorrectly. In that case it'd make more sense that we raised a more helpful error than what we might get from braintree.